src/VscPreviewExtension/package.json

```diff
@@ -1,21 +1,25 @@
 {
   "name": "DfmPreview",
   "displayName": "DfmPreview",
-  "description": "Preview the DocFX Flavored Markdown in Visual Studio Code",
-  "version": "0.0.9",
+  "description": "Preview the DocFX Flavored Markdown and tokenTree in Visual Studio Code",
+  "version": "0.0.15",
   "publisher": "928pjy",
   "engines": {
     "vscode": "^1.3.0"
   },
   "categories": [
+    "Languages",
     "Other"
   ],
   "activationEvents": [
     "onCommand:DFM.showPreview",
     "onCommand:DFM.showPreviewToSide",
-    "oncommand:DFM.showSource"
+    "oncommand:DFM.showSource",
+    "onCommand:DFM.showTokenTreeToSide"
   ],
   "main": "./out/src/extension",
+  "icon": "img/dfm.png",
```

Comment:
Better update this time, Just in case I forget

---
In reply to: [77451876](https://github.com/dotnet/docfx/pull/664#discussion_r77451876) [](ancestors = 77451876,77431307,77354251)